src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdownModalContent.ts

```diff
@@ -0,0 +1,45 @@
+import { IResponsiveDropdownContent, ResponsiveDropdownContent } from './ResponsiveDropdownContent';
+import { Dom, $$, l } from '../../../Core';
+import { AccessibleButton } from '../../../utils/AccessibleButton';
+
+export class ResponsiveDropdownModalContent implements IResponsiveDropdownContent {
+  private closeButton: Dom;
+  private className: string;
+
+  private set hidden(shouldHide: boolean) {
+    this.element.toggleClass('coveo-hidden', shouldHide);
+  }
+
+  constructor(private componentName: string, public element: Dom, private closeButtonLabel: string, private close: () => void) {
+    this.className = `coveo-${this.componentName}-dropdown-modal-content`;
+  }
+
+  public positionDropdown(): void {
+    this.element.el.classList.add(this.className, ResponsiveDropdownContent.DEFAULT_CSS_CLASS_NAME);
+    this.element.setAttribute('role', 'group');
+    this.element.setAttribute('aria-label', l('FiltersDropdown'));
+    this.hidden = false;
+    this.closeButton = $$('div', { className: 'coveo-facet-modal-close-button' }, '✕');
```

Comment:
Is there an existing `X` icon that we could reuse for consistency across the components? For example, the `X` we display to reset a facet with active values?

Reply:
You're right, I'll use an icon instead.

Reply:
yeah, I'm pretty sure it's available as an SVG

Reply:
Also, I'm seeing we create a `div` and then make it accessible by wrapping it with the accessible button class.

Is there a reason we don't skip all of this and just use a `button` element ?

Reply:
You're right, I'll make this a button instead.

I tend to avoid buttons by reflex because of their default styling, but for accessibility purposes it's definitely cleaner to make it a button in the first place.

Reply:
If that doesn't already exist, you could create a sass mixin that removes a button ugly default styles 😄

Reply:
Hmm, that's tempting. Although in this case, the only property I had to explicitly remove was  the `border`.